azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeCallScreenHeaderOptions.java

```diff
@@ -3,12 +3,15 @@
 /* <CUSTOM_CALL_HEADER> */
 package com.azure.android.communication.ui.calling.models;
 
+import com.azure.android.communication.ui.calling.presentation.manager.CallScreenInfoHeaderManager;
+
 /**
  * Options for the CallCompositeCallScreenHeaderOptions.
  */
 public final class CallCompositeCallScreenHeaderOptions {
-    private CallCompositeCallDurationTimer timer;
+    CallScreenInfoHeaderManager callScreenInfoHeaderManager;
```

Comment:
I would consider to not to include a manager in the Options class.
I would expose an event, then I would instead of assigning instance  a manager, I would subscribe on event.
The reason for this is a single responsibility principle. I like that Options class remains a data class only. When we do a composition with a manager class it now is responsible for some behaviours. Why not to keep Options class simple.

Reply:
I thin we can use interface, instead of setting multiple event handlers.